app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -522,6 +542,11 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
     $scope.angularForm[$scope.authType + '_auth_status'].$setViewValue(updatedValue);
   };
 
+  $scope.updateHawkularHostname = function(value) {
+    $scope.angularForm['hawkular_hostname'].$setViewValue(value);
```

Comment:
You probably don't even need`$setViewValue`. 
Just this `$scope.emsCommonModel.hawkular_hostname = value` should work.